server/data/hmppsAuthClient.test.ts

```diff
@@ -97,7 +90,7 @@ describe('hmppsAuthClient', () => {
       const output = await hmppsAuthClient.getSystemClientToken()
 
       expect(output).toEqual(token.access_token)
-      expect(redis.set).toBeCalledWith('Bob', token.access_token, 'EX', 240, expect.any(Function))
+      expect(tokenStore.setToken).toBeCalledWith('%ANONYMOUS%', token.access_token, 240)
```

Comment:
this test was previously incorrect as mocking wasn't working properly - the token should be stored against anonymous